packages/typespec-ts/test/integration/generated/azure/core/src/models.ts

```diff
@@ -22,3 +22,6 @@ export interface ListItemInputBody {
   /** The name of the input. */
   inputName: string;
 }
+
+/** Alias for ListItemInputExtensibleEnum */
+export type ListItemInputExtensibleEnum = string | "First" | "Second";
```

Comment:
this is because cadl-ranch has changed extensible enum into union, I think we could support it better when we finish the extensible enum review.